joern-cli/frontends/jimple2cpg/src/main/scala/io/joern/jimple2cpg/passes/AstCreator.scala

```diff
@@ -993,4 +975,73 @@ object AstCreator {
       f(x, i + 1)
     }.toSeq
   }
+
+  private val PRIMITIVES: Map[Char, String] = Map[Char, String](
```

Comment:
```suggestion
  private val Primitives: Map[Char, String] = Map[Char, String](
```
scala devs don't tend to SHOUT :)
up to you though, this may have some meaning, and is really just a style guide, not a mandate

Reply:
Lol, got it. Will make suggested change.